lib/IDE/CodeCompletion.cpp

```diff
@@ -118,7 +118,7 @@ class CodeCompletionCallbacksImpl : public CodeCompletionCallbacks {
   /// In situations when \c SyntaxKind hints or determines
   /// completions, i.e. a precedence group attribute, this
   /// can be set and used to control the code completion scenario.
-  SyntaxKind SyntxKind;
+  SyntaxKind SyntaxKind_;
```

Comment:
The obvious spelling fix would collide w/ the type. Offhand, this abbreviation doesn't appear to be used by this project. I'm open to suggestions. `SK` is taken below and `Kind` above.